### PR TITLE
Show last score and high score

### PR DESCRIPTION
**Pull request by tjorim**
## Description

This will show and save the highscore,
as well as the score from last game.

There were also multipe generalStyle variables which have been put together
and not-closed `<span>`-tags are fixed now.
## Motivation and Context

This will partially fix issue #38 (more some sort of feature)
and help with the implementation of #129 .

The code could be adjusted to save multiple score
in an array with day and maybe bot version.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: bot.user.js [hunk 1]
```diff
@@ -1498,35 +1520,43 @@ window.sosBackup = sos;
...
+    window.changeskin_overlay.innerHTML = window.spanstyle +
         '(X) Change skin </span>';
-    window.quickResp_overlay.innerHTML = generalStyle +
+    window.quickResp_overlay.innerHTML = window.spanstyle +
         '(ESC) Quick Respawn </span>';
     // eslint-disable-next-line no-undef
-    window.version_overlay.innerHTML = generalStyle + 'Version: ' + GM_info
-        .script.version;
+    window.version_overlay.innerHTML = window.spanstyle +
+        'Version: ' + GM_info.script.version + '</span>';
```
Comment (tjorim) [inline bot.user.js:1533]: This is a Greasemonkey variable, ignore.

File: bot.user.js [hunk 2]
```diff
@@ -436,17 +437,14 @@ var bot = (function() {
...
                 bot.connectBot();
-                if (document.querySelector('div#lastscore').childNodes
-                    .length > 1) {
-                    window.scores.push(parseInt(document.querySelector(
-                        'div#lastscore').childNodes[1].innerHTML));
-                }
+                /* The lastscore should be checked here in the future,
+                but not only when bot, just when not playing.
+                THIS SHOULD ONLY BE EXECUTED ONCE! */
+                userInterface.saveScore(); // Checks highscore
```
Comment (tjorim) [inline bot.user.js:434]: Should I just move the whole userInterface-module up?
Or is there something like function prototyping in C?

File: bot.user.js [hunk 3]
```diff
@@ -1498,35 +1521,44 @@ window.sosBackup = sos;
...
         '(X) Change skin </span>';
-    window.quickResp_overlay.innerHTML = generalStyle +
+    window.quickResp_overlay.innerHTML = window.spanstyle +
         '(ESC) Quick Respawn </span>';
     // eslint-disable-next-line no-undef
-    window.version_overlay.innerHTML = generalStyle + 'Version: ' + GM_info
-        .script.version;
-
+    window.version_overlay.innerHTML = window.spanstyle +
+        'Version: ' + GM_info.script.version + '</span>';
```
Comment (tjorim) [inline bot.user.js:1533]: Again, this is a Greasemonkey variable, ignore.

File: bot.user.js (hunk 2 above)
Comment (ChadSki) [inline bot.user.js:434]: Hmm... does it run correctly in the browser?

Comment (tjorim) [inline bot.user.js:434]: @ChadSki Looks like it does yeah.

**Comment by ermiyaeskandary** [comment]
@tjorim 
`"greasemonkey": true` is an invalid option - conflicts ^

**Comment by tjorim** [comment]
@ErmiyaEskandary Strange, it should work: http://eslint.org/docs/user-guide/configuring
Anyway, I reverted that last commit.

**Comment by ermiyaeskandary** [comment]
Approved - conflicts ;)

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/183/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by ChadSki** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/183/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by ermiyaeskandary** [comment]
@tjorim Resolve conflicts please and then we can merge...

**Comment by ermiyaeskandary** [comment]
@tjorim Should work...

File: bot.user.js [hunk 4]
```diff
@@ -2122,10 +2127,10 @@ var collisionGrid = (function() {
...
            // y = y - (y % collisionGrid.cellSize);
 
-            col = parseInt(Math.floor(x / collisionGrid.cellSize));
-            row = parseInt(Math.floor(y / collisionGrid.cellSize));
-            col = Math.min(Math.max(col, 0), collisionGrid.gridWidth);
-            row = Math.min(Math.max(row, 0), collisionGrid.gridHeight);
+            var col = parseInt(Math.floor(x / collisionGrid.cellSize));
+            var row = parseInt(Math.floor(y / collisionGrid.cellSize));
+            var col = Math.min(Math.max(col, 0), collisionGrid.gridWidth);
+            var row = Math.min(Math.max(row, 0), collisionGrid.gridHeight);
```
Comment (ChadSki) [inline bot.user.js:2133]: Don't need `var` the 2nd time

**Pull request closed without merging**